### PR TITLE
Fix `previous-next-commit-buttons` on condensed commit views

### LINKED ISSUE #4503: misaligned commit navigation on 'condensed' merge commit
the prev / next navigation buttons at the bottom of commits are misaligned on a merge commit that is 'condensed' ( possibly merge commit after conflicts ). GitHub shows a warning notice saying that the view is _a condensed version of th[e] merge commit_. But the commit navigation overlaps the warning and 'sticks' to top right border of the warning ( the stickiness probably has nothing to do with the warning because it is absolutely positioned ).

image

[source commit](https://github.com/sindresorhus/refined-github/pull/4448/commits/0b8966c918eae11da9fc992368741757088edf08)

### PR DESCRIPTION
**Pull request by cheap-glitch**
Fixes #4503 

## Test URLs

* [Condensed commit](https://togithub.com/refined-github/refined-github/pull/4448/commits/0b8966c918eae11da9fc992368741757088edf08)
* [Regular commit](https://togithub.com/refined-github/refined-github/pull/5113/commits/5b7282afc40b013f5928271fb6740cf70b4e4295)

## Screenshot

**Before**

image

**After**

image

### CODE REVIEW

File: source/features/previous-next-commit-buttons.tsx [hunk 1]
```diff
@@ -9,10 +10,14 @@ function init(): false | void {
...
-	const files = select('#files')!;
+	// Wrap the button in a <div> to avoid #4503
+	select('#files')!.after(
+		<div className="d-flex flex-justify-end">
+			{originalPreviousNext.cloneNode(true)}
+		</div>,
+	);
 
-	files.after(previousNext);
+	select('#files ~ .flash-warn')?.classList.add('mt-3');
```
Comment (cheap-glitch) [inline source/features/previous-next-commit-buttons.tsx:20]: Not sure if the class should be applied to the wrapping `div` instead.

Comment (fregante) [reply source/features/previous-next-commit-buttons.tsx:20]: If it works there yes. It shortens this file by 2 lines right?

**Comment by fregante** [comment]
We could add it unconditionally. Is that too much whitespace?

**Comment by cheap-glitch** [comment]
> Is that too much whitespace?

Not really imo, but the difference is visible.

<table>
<tr>
	<td>Before
	<td>After
<tr>
	<td>

screenshot-1638889378
	<td>

screenshot-1638889343
</table>

**Comment by fregante** [comment]
👆 LGTM

**Pull request merged**